include/roboteam_utils/BiMap.h

```diff
@@ -69,13 +69,13 @@ namespace rtt {
          * @return false If key is not found
          */
         bool removeByLeft(const T &key) {
-            auto it = leftSet.find({left, nullptr});
+            auto it = leftSet.find({key, nullptr});
```

Comment:
This entire structure is basically lookup in sets, meaning it's an std::map

Reply:
yeah.. It's unused so if it's up to me we just remove the class (as it also actually throws a compile error when you try to instantiate the template)

Reply:
Lets remove it. We can always reimplement it if needed